client/components/open/forms/components/FormSubmissions.vue

```diff
@@ -42,6 +42,13 @@
             </VForm>
           </div>
           <div class="font-semibold flex gap-2">
+            <USelectMenu
+              v-if="form.enable_partial_submissions"
+              v-model="selectedStatus"
+              :options="statusList"
+              value-attribute="value"
+              option-attribute="label"
+            />
             <UButton
```

Comment:
![CleanShot 2025-02-24 at 16 41 59@2x](https://github.com/user-attachments/assets/3e2802e1-15e7-49ea-b2a5-79f4683c3bcf)
Display issue - please give fixed with that fits both labels